### PR TITLE
[CLC-169] Add jet job tutorial: updates

### PR DESCRIPTION
**Pull request by yuce**
This PR updates: https://github.com/hazelcast/hazelcast-commandline-client/pull/242 with a Java based project.

### CODE REVIEW

File: docs/modules/ROOT/pages/jet-job-management.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+== Step 4. Cancel the Jet job
+
+Now, cancel the Jet job.
+
+[source,shell]
+----
+clc -c dev job cancel prime-job
+----
+
+Observe that the size of `my-map` doesn't increase anymore.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-job-management.adoc:91]:
```suggestion
. See how the size of the `my-map` remains the same.
```

File: docs/modules/ROOT/pages/jet-job-management.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  clc -c dev map size -n my-map; sleep 3; \
+done
+----
+
+== Summary
+
+In this tutorial, you learned how to do the following:
+
+* Connect to a Hazelcast cluster.
+* Build and submit Hazelcast Jet pipelines.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-job-management.adoc:105]:
```suggestion
* Build and submit Hazelcast Jet jobs.
```

File: docs/modules/ROOT/pages/jet-job-management.adoc [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+* Manage Jet pipeline lifecycle with commands: list and cancel.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-job-management.adoc:106]:
```suggestion
* Manage the lifecycle of a Jet pipeline using list and cancel commands.
```

**Review by angelasimms — COMMENTED** [review]
I've left a few comments. These are more formatting than anything else.

**Review by angelasimms — COMMENTED** [review]
I've left a few comments. These are more formatting than anything else.

File: docs/modules/ROOT/pages/jet-job-management.adoc [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+--
+====
+
+[[step-2-build-jet-job]]
+== Step 2. Build the Jet Job
+
+The sample Jet job we use in this tutorial generates a stream of numbers and creates a link:https://hashids.org/[Hashid] for each number. The number and hashid pair is then saved to a Map with they number as the key and hashid as the value.
+
+You can find the Jet job project in https://github.com/hazelcast/hazelcast-commandline-client[the CLC repository] under the path `examples/platform/simple-streaming-pipeline`.
+
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/ROOT/pages/jet-job-management.adoc:62]: I'll merge this PR without this step, but but will add that after releasing v5.3.0

File: docs/modules/ROOT/pages/jet-job-management.adoc [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+Build the project with the following command:
+
+[source,shell]
+----
+gradle shadowJar
+----
+
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/ROOT/pages/jet-job-management.adoc:69]: I'll merge this PR without this output, but but will add that after releasing v5.3.0

**Comment by yuce** [comment]
@angelasimms I've updated the PR with your suggestions (with a few exceptions) and will merge it.
We can fix any problems with the doc after v5.3.0 release.

**Pull request merged**